engine/src/main/java/org/terasology/engine/logic/debug/ChunkEventErrorLogger.java

```diff
@@ -25,14 +25,14 @@ public class ChunkEventErrorLogger extends BaseComponentSystem {
     @ReceiveEvent(components = WorldComponent.class)
     public void onNewChunk(OnChunkLoaded chunkAvailable, EntityRef worldEntity) {
         if (!loadedChunks.add(chunkAvailable.getChunkPos())) {
-            logger.error("Multiple loads of chunk {}", chunkAvailable.getChunkPos());
+            logger.atError().addArgument(() -> chunkAvailable.getChunkPos()).log("Multiple loads of chunk {}");
```

Comment:
simple getter method so impact neglectable
```suggestion
            logger.error("Multiple loads of chunk {}", chunkAvailable.getChunkPos()); //NOPMD
```